Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/FingerCursor.cs

```diff
@@ -118,14 +124,21 @@ protected override void UpdateCursorTransform()
 
                     if (indexFingerRingRenderer != null)
                     {
-                        TranslateToFinger(indexFingerRingRenderer.transform, deltaTime, indexFingerPosition, indexFingerRotation, nearGrabbable);
+                        if (nearGrabbable)
```

Comment:
Does anyone know if this unique behaviour for grab is desired? Also, is the thumb cursor needed?

Reply:
@Cameron-Micka  I believe this was for an experiment, and can be removed (no need for thumb cursor)

Reply:
Yes, this was for a experiment if we wanted a cursor on the thumb when grabbing. I noticed the shell still has (dead) code paths for this. So, maybe it will come back at some point?